src/networking.c

```diff
@@ -1181,28 +1217,32 @@ void AddReplyFromClient(client *dst, client *src) {
      * for some reason the output limits don't reach the same decision (maybe
      * they changed) */
     if (src->flags & CLIENT_CLOSE_ASAP) {
-        sds client = catClientInfoString(sdsempty(),dst);
+        sds client = catClientInfoString(sdsempty(), dst);
         freeClientAsync(dst);
-        serverLog(LL_WARNING,"Client %s scheduled to be closed ASAP for overcoming of output buffer limits.", client);
+        serverLog(LL_WARNING, "Client %s scheduled to be closed ASAP for overcoming of output buffer limits.", client);
         sdsfree(client);
         return;
     }
 
     /* First add the static buffer (either into the static buffer or reply list) */
-    addReplyProto(dst,src->buf, src->bufpos);
+    addReplyProto(dst, src->buf, src->bufpos);
 
     /* We need to check with prepareClientToWrite again (after addReplyProto)
      * since addReplyProto may have changed something (like CLIENT_CLOSE_ASAP) */
-    if (prepareClientToWrite(dst) != C_OK)
+    if (prepareClientToWrite(dst) != C_OK) {
         return;
+    }
 
     /* We're bypassing _addReplyProtoToList, so we need to add the pre/post
      * checks in it. */
-    if (dst->flags & CLIENT_CLOSE_AFTER_REPLY) return;
+    if (dst->flags & CLIENT_CLOSE_AFTER_REPLY) {
+        return;
+    }
 
     /* Concatenate the reply list into the dest */
-    if (listLength(src->reply))
-        listJoin(dst->reply,src->reply);
+    if (listLength(src->reply)) {
+        listJoin(dst->reply, src->reply);
+    }
```

Comment:
It feels like every other change is this. I feel strongly now that we shouldn't enforce that everything is wrapped, because we can skip it right?